pkg/chart/loader/load.go

```diff
@@ -205,3 +206,46 @@ func LoadFiles(files []*BufferedFile) (*chart.Chart, error) {
 
 	return c, nil
 }
+
+// LoadValues loads chat values from a reader.
+func LoadValues(data io.Reader) (map[string]interface{}, error) {
+	values := map[string]interface{}{}
+	reader := utilyaml.NewYAMLReader(bufio.NewReader(data))
+	for {
+		currentMap := map[string]interface{}{}
+		raw, err := reader.Read()
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+			return nil, errors.Wrap(err, "error reading yaml document")
+		}
+		if err := yaml.Unmarshal(raw, &currentMap, func(d *json.Decoder) *json.Decoder {
+			d.UseNumber()
+			return d
+		}); err != nil {
+			return nil, errors.Wrap(err, "cannot unmarshal yaml document")
+		}
+		values = MergeMaps(values, currentMap)
+	}
+	return values, nil
+}
+
```

Comment:
This should technically have a doc string too since it is exported.

Reply:
thanks, also updated here